packages/frontend/amp/components/AdConsent.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import { AMPJSON } from './AMPJSON';
+import { css, cx } from 'emotion';
+import Tick from '@guardian/pasteup/icons/tick.svg';
+
+const fontFamily =
+    'GuardianTextSans,Helvetica Neue,Helvetica,Arial,Lucida Grande,sans-serif';
+
+const consentUIStyle = css`
+    font-family: ${fontFamily};
+    font-size: 1rem;
+    line-height: 1.25rem;
+    color: #f6f6f6;
+    background-color: rgb(51, 51, 51);
+
+    max-width: 37.5rem;
+    margin: 0 auto;
+    overflow-x: hidden;
+`;
+
+const containerDivStyle = css`
+    margin: 1em;
+`;
+
+const h2Style = css`
+    margin-bottom: 1em;
+    font-size: 1.5em;
+    font-weight: bold;
+`;
+
+const pStyle = css`
+    margin-bottom: 0.5em;
+`;
+
+const aStyle = css`
+    color: #f6f6f6;
+    &:hover {
+        text-decoration: 0;
```

Comment:
Should this be `text-decoration: none`? `0` isn't a valid value for `text-decoration`.

Reply:
I thought so too, until the frontend linter told me otherwise :)

Reply:
It's weird that you saw that. The linter doesn't lint properties CSS in JS, just syntax. Perhaps it's the way your editor is configured? Either way, can you try changing it to `none` and running `make lint`?